.cbmc-batch/jobs/aws_cryptosdk_rsa_encrypt/aws_cryptosdk_rsa_encrypt_harness.c

```diff
@@ -21,31 +21,39 @@
 #define KEY_LEN 256
 
 void aws_cryptosdk_rsa_encrypt_harness() {
+    /* Nondet Inputs */
     struct aws_byte_buf cipher;
     struct aws_allocator *alloc = can_fail_allocator();
     struct aws_byte_cursor plain;
-    struct aws_string *key = ensure_string_is_allocated_bounded_length(KEY_LEN);
+    struct aws_string *key = ensure_string_is_allocated_nondet_length();
     enum aws_cryptosdk_rsa_padding_mode rsa_padding_mode;
 
+    /* Assumptions */
+    __CPROVER_assume(aws_string_is_valid(key));
+    __CPROVER_assume(key->len <= KEY_LEN);
```

Comment:
We should have a "bounded" allocated nondet length predicate, so we don't have to assume the bound here (but this is not a blocker for this PR).

Reply:
Yeah I've had to do this in other PRs to so let me make a github issue about it in c-common.